pkg/backup/backupdest/incrementals.go

```diff
@@ -191,23 +275,25 @@ func ResolveIncrementalsBackupLocation(
 		// knows this isn't a usable incrementals store.
 		// Some callers will abort, e.g. BACKUP. Others will proceed with a
 		// warning, e.g. SHOW and RESTORE.
-		_, err = backupsFromLocation(ctx, user, execCfg, incPaths[0])
+		_, err = backupsFromLocation(
+			ctx, user, execCfg, defaultCollectionURI, subdir, incPaths[0], true, /* customIncLocation */
+		)
 		if err != nil {
 			return nil, err
 		}
 		return incPaths, nil
 	}
-
 	resolvedIncrementalsBackupLocation, err := backuputils.AppendPaths(fullBackupCollections, backupbase.DefaultIncrementalsSubdir, subdir)
 	if err != nil {
 		return nil, err
 	}
-
-	_, err = backupsFromLocation(ctx, user, execCfg, resolvedIncrementalsBackupLocation[0])
+	_, err = backupsFromLocation(
+		ctx, user, execCfg, defaultCollectionURI, subdir,
+		resolvedIncrementalsBackupLocation[0], false, /* customIncLocation */
```

Comment:
nit, i'd instead define `customIncLocation := len(explicitIncrementalCollections) > 0 ` above the branch and pass that instead of hard coded bools. That protects against a future bug where someone monkeys around with the flow control here.